src/app/api/routes/devices.py

```diff
@@ -140,6 +149,20 @@ async def update_my_location(
     return device
 
 
+@router.put("/hash", response_model=DeviceOut, summary="Update the software hash of the current device")
+async def update_my_hash(
+    payload: SoftwareHash, device: DeviceOut = Security(get_current_device, scopes=["device"])
+):
+    """
+    Updates the location of the current device
```

Comment:
Just need to modify the docstring I think.

This gives to the device the latest hash it should have right ? Then the device will have to check if the hash it had before calling this method is the same as the new one ?

Reply:
My bad, I got confused: the device itself should not be able to update its expected hash. It should only fetch the expected hash and check whether it's up-to-date. I'll fix this